app/graphql/types/query_type.rb

```diff
@@ -20,10 +20,18 @@ def verse(id:)
       Verse.find(id)
     end
 
-    field :verses, [Types::VerseType], null: false 
+    field :verses, [Types::VerseType], null: false
```

Comment:
BREAKING CHANGES

Corporate want you to find the difference between these two lines

(They're the same line)